README.md

```diff
@@ -424,6 +424,33 @@ const handleRequest = (req, res) => {
 
 Make sure you replace the static hook with the real one when you hydrate your app on a client.
 
+If you want to be able to detect redirects you can provide the `record` option:
+
+```js
+import { renderToString } from "react-dom/server";
+import { Router } from "wouter";
+import staticLocationHook from "wouter/static-location";
+
+import App from "./app";
+
+const handleRequest = (req, res) => {
+  const location = staticLocationHook(req.path, { record: true });
+  const prerendered = renderToString(
+    <Router hook={location}>
+      <App />
+    </Router>
+  );
+
+  // location.history is an array matching the history a
+  // user's browser would capture after loading the page
+
+  const finalPage = locationHook.history.slice(-1)[0];
```

Comment:
I wonder if there more clear way to get the last element of the array. For some beginner readers of the README this might look confusing. 
I was thinking we could do that with array destructuring first, but then it appeared it's not possible with ES6. 

Any ideas?

Reply:
Well it could always be added as a dedicated property on the hook for convenience. Size doesn't really matter so much for this code because it's only going to be server-side / dev-side, but I didn't want to bloat the API. So if you'd prefer, it could be added as `locationHook.current` or something.

Reply:
I think it's going to do the job 😄 Let's merge the PR.